build.cake

```diff
@@ -5,14 +5,22 @@
 // ARGUMENTS
 //////////////////////////////////////////////////////////////////////
 
+const string DefaultVersion = "1.0.0.0";
+
 var target = Argument("build-target", "Default");
-var version = Argument("build-version", "1.0.0.0");
+var version = Argument("build-version", DefaultVersion);
 var configuration = Argument("build-configuration", "Release");
 
+if (version == DefaultVersion)
+    Warning($"No version specified. Using default version '{DefaultVersion}'.");
```

Comment:
Redundant.

Reply:
Why?

Reply:
Because if you don't specify `X`, either one of 2 things will happen, it will fail, or it will use the default value.

I find it rather redundant to say "using the default version".